pkg/connectors/datacatalog/clients/datacatalog_openapi.go

```diff
@@ -15,6 +16,13 @@ import (
 	"fybrik.io/fybrik/pkg/model/datacatalog"
 )
 
+// ErrorMessages that are reported to the user
+const (
+	InvalidAssetID        string = "the asset does not exist"
```

Comment:
why not use the specific causein the name?  InvalidAssetID -> AssetIDNotFound? invalid is general...

Reply:
Changed.